src/pages/SpamFilterSettings.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Flex, Box, Text, Button } from '@chakra-ui/react';
+
+import MainViewLayout from '../components/mainViewLayout';
+import useBoost from '../hooks/useBoost';
+import { getReadableBalance } from '../utils/tokenValue';
+import { useAppModal } from '../hooks/useModals';
+import { BOOSTS } from '../data/boosts';
+import { FIELD } from '../data/fields';
+
+const prepareFormLegoForUpdate = () => {
```

Comment:
Whoa, yeah this seems odd. I'd be down to discuss this pattern further once I understand it.

Reply:
decoded to move this to a new lego. BOOST_SETTINGS in a new data/boostSetting.js file. Can still use boost legos and override things.